prow/e2e-suite.sh

```diff
@@ -82,5 +82,5 @@ if ${SINGLE_MODE}; then
 
 else
     echo "Executing e2e test suite"
-    time ISTIO_DOCKER_HUB=$HUB E2E_ARGS="${E2E_ARGS[@]}" make e2e_all
+    time ISTIO_DOCKER_HUB=$HUB E2E_ARGS="${E2E_ARGS[@]}" make e2e_all >&1 | go-junit-report > junit.xml
```

Comment:
I'm unclear about what is accomplished by the ">&1".  I'd understand "2>&1" but this seems to pipe stdout to itself.  It this a no-op or does it have some other side effect?  I thought this might be to allow stdout to be both output and captured but that's not what happened for me and instead I had to resort to something like:
<pre>
make e2e_all > tee >(go-junit-report > junit.xml)
</pre>

Reply:
That was my intention to log to console and to stream through pipe. you are right, `>&1` really does nothing since the log already prints to stdout by default, which is then dup-ed to whichever file descriptor console/tty is at. What I had was essentially redirecting stdout to stdout... Good catch. I have patched with your fix.